src/org/opensolaris/opengrok/search/Results.java

```diff
@@ -171,7 +175,18 @@ public static void prettyPrint(Writer out, SearchHelper sh, int start,
                 out.write(rpathE);
                 out.write("\">");
                 out.write(rpath.substring(rpath.lastIndexOf('/') + 1)); // htmlize ???
-                out.write("</a></td><td><tt class=\"con\">");
+                out.write("</a>");
+                if(sh.lastEditedDisplayMode){
+                    try {
+                        // insert last edited date if possible
+                        df = DateFormat.getDateTimeInstance(DateFormat.SHORT, DateFormat.SHORT);
+                        String dd = df.format(DateTools.stringToDate(doc.get("date")));
+                        out.write("<small class=\"edited\">(last edited ");
+                        out.write(dd);
+                        out.write(")</small>");    
+                    } catch (ParseException ex) {}                    
```

Comment:
it would make sense to add a log output, though it would popup in webapp container which are usually set to log only warning and above so not sure if it will help, but it will definitely help people who know how to setup loggers but lack info how to debug ...
so yes, log would be cool though not mandatory(I mean, if the <small won't show it means there was some exception, so you would have to increase the logger level or debug anyways, which from my POV is the same effort ;) )